homeassistant/helpers/intent.py

```diff
@@ -493,14 +494,21 @@ async def async_handle_states(
     async def async_call_service(self, intent_obj: Intent, state: State) -> None:
         """Call service on entity."""
         hass = intent_obj.hass
-        await hass.services.async_call(
-            self.domain,
-            self.service,
-            {ATTR_ENTITY_ID: state.entity_id},
-            context=intent_obj.context,
-            blocking=True,
-            limit=self.service_timeout,
+        task = hass.async_create_task(
+            hass.services.async_call(
+                self.domain,
+                self.service,
+                {ATTR_ENTITY_ID: state.entity_id},
+                context=intent_obj.context,
+                blocking=False,
+            )
         )
+        # Block with a short timeout to (hopefully) catch validation errors
+        try:
+            async with async_timeout.timeout(self.service_timeout):
```

Comment:
also `asynico.wait`

Reply:
Actually, I noticed this code has `blocking=False`. This makes me think this can be removed.

Reply:
Fixed.